providers/bulkSms/src/lib/bulkSms.provider.ts

```diff
@@ -0,0 +1,35 @@
+import { ChannelTypeEnum, ISendMessageSuccessResponse, ISmsOptions, ISmsProvider } from '@novu/stateless';
+
+export class BulkSmsSmsProvider implements ISmsProvider {
+  id = 'bulksms';
+  channelType = ChannelTypeEnum.SMS as ChannelTypeEnum.SMS;
+
+  constructor(
+    private config: {
+      authKey?: string;
+      sender?: string;
+      route?: string;
+      // dltEntityId?: string;
+    }
+  ) {}
+
+  async sendMessage(options: ISmsOptions): Promise<ISendMessageSuccessResponse> {
+    console.log(options);
+    const response1 = await fetch(
+      // I will put my API URL here so that we can send sms to our clients (Ex - 'http://login.bulksmsoffers.com/'),
+      {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      }
+    );
+    const data = await response1.json();
+    console.log(data);
+
+    return {
+      id: 'yymd',
+      date: '20221027',
+    };
+  }
+}
```

Comment:
Is this section is Work in progress?

Reply:
I have just given an example that how bulksms is working, here we just need to hit the API that bulksms has provided us and we need to track the response accordingly. For testing purpose I have added this in the mailjet provider i.e we will call mailjet provide but inside that send message I have given the API of bulksms and it is working fine. For your reference I am attaching the screenshot.
![image](https://user-images.githubusercontent.com/94128805/200998714-2190e805-fc46-468f-9514-43557f5681aa.png)

Reply:
I consider that the right place to show an example of how this provider would work is either documentation or in the tests. This file needs to be the integration with BulkSMS in a generic way so anyone can use it.

Reply:
Okay

Reply:
This hasn't been addressed yet.

Reply:
@p-fernandez Have done certain chnages but the URL and the return part will be managed by the users because every user will be having different URL which will be provided by the Bulksms team so I can't prepare a particular model which will be used by everyone but yes the URL link I have mentioned people can put there URL there and they can use it. The bulksms is little bit different than other SMS providers. We can also make it dynamic i.e if we will pass the bulksms url in the frontend then we can fetch it and use it here.

Reply:
If the users need to provide a certain URL provided by BulkSMS then we would need to add to the config the ability for the users to pass that URL and use it in this file.
I am sorry to say but as it is now this file is not usable, and neither the provider module.